support-files/mysql-multi.server.sh

```diff
@@ -107,7 +107,7 @@ fi
 datadir=@localstatedir@
 basedir=
 pid_file=
-parse_arguments `$print_defaults $defaults mysqld mysql_server mysql_multi_server`
+parse_arguments `$print_defaults $defaults mariadbd mysqld mysql_server mysql_multi_server`
```

Comment:
If not, then [mysqld] section would not be read? I am not an expert on how my_print_default works.

Reply:
Yes, [mysqld] needs to stay in the list as even our default configs define stuff for `[mysqld]`.

The `[mariadbd]` I think is a good addition in case there is some new option only `mariadbd`can run. The `my_print_defaults` command already parses these correctly, tested it with:

```
/etc/mysql/mariadb.cnf:

[mariadbd]
test-mariadbd = true

[mysqld]
test-mysqld = true


root@a71a6e9c1776:/etc/mysql# my_print_defaults mariadbd mysqld                                
--test-mariadbd=true
--test-mysqld=true
..

root@a71a6e9c1776:/etc/mysql# my_print_defaults mysqld
--test-mysqld=true
...

root@a71a6e9c1776:/etc/mysql# my_print_defaults mariadbd
--test-mariadbd=true

root@a71a6e9c1776:/etc/mysql# my_print_defaults mysqld  
--test-mysqld=true
--user=mysql
--pid-file=/run/mysqld/mysqld.pid
--basedir=/usr
--datadir=/var/lib/mysql
--tmpdir=/tmp
--lc-messages-dir=/usr/share/mysql
--lc-messages=en_US
--skip-external-locking
--bind-address=127.0.0.1
--expire_logs_days=10
--character-set-server=utf8mb4
--collation-server=utf8mb4_general_ci
```